oidc-appsupport/src/iosMain/kotlin/org/publicvalue/multiplatform/oidc/appsupport/IosCodeAuthFlowFactory.kt

```diff
@@ -15,11 +14,14 @@ class IosCodeAuthFlowFactory(
     private val ephemeralBrowserSession: Boolean = false,
     /** factory used to create preferences to save session information during login process. **/
     private val preferencesFactory: PreferencesFactory = PreferencesFactory()
-): CodeAuthFlowFactory {
-    private val preferences = runBlocking { preferencesFactory.getOrCreate(PREFERENCES_FILENAME) }
+) : CodeAuthFlowFactory {
+    private val preferences = preferencesFactory.create(PREFERENCES_FILENAME)
```

Comment:
Assuming the factory has already been created, this would crash.
Assuming it was not created, the only thing that is saved with this change should be the lock - is that really impacting login time?